pkg/descheduler/strategies/lownodeutilization_test.go

```diff
@@ -52,23 +50,6 @@ func setHighPriority(pod *v1.Pod)        { pod.Spec.Priority = &highPriority }
 func setLowPriority(pod *v1.Pod)         { pod.Spec.Priority = &lowPriority }
 func setNodeUnschedulable(node *v1.Node) { node.Spec.Unschedulable = true }
```

Comment:
There's nothing specific to LowNodeUtilization strategy about these helper functions. You can move all of them under `test/test_utils.go` as well.